language-support/java/bindings-rxjava/BUILD.bazel

```diff
@@ -51,6 +51,7 @@ da_java_library(
 )
 
 testDependencies = [
+    "//daml-lf/data",
```

Comment:
Is this dependency still used?

Reply:
Yes, to create a toke with a `Ref.Party` in a claim for testing.

Reply:
👍